### ISSUE TITLE
ACF accordion addon breaks after updating to ACF v. 5.6.6

### ISSUE THREAD

**Issue by bhuisman1992** [bvdr/acf-accordion#11]
Hi there,

ACF accordion addon breaks after updating to ACF v. 5.6.6. I get the following error

![schermafdruk 2017-11-28 09 56 15](https://user-images.githubusercontent.com/22633252/33310842-41670800-d423-11e7-8936-9bc819b6ec20.png)

I'm running ACF PRO version 5.6.6 and the latest ACF accordion version in combination with WordPress 4.9

**Comment by bvdr** [comment]
Hi @bhuisman1992, 

Thank you for letting me know.
Haven't got time to maintain the plugin.
Hope I find some time to fix this.

Regards,
Bogdan

**Comment by bvdr** [comment]
From what it looks you have the accordion plugin loaded two times.
Please check the plugins folder and make sure you delete `/acf-accordion.deactivate` folder first.

**Comment by bhuisman1992** [comment]
Well, the funny thing is.. i am not

**Comment by bhuisman1992** [comment]
Tested it on a other website.. getting the same results with only one plugin installed

**Comment by bvdr** [comment]
OK, :)

Thanks for reporting!

**Comment by bhuisman1992** [comment]
Any change you can fix this? Seems like me it's a small fix!

**Comment by bvdr** [comment]
I'll work on a fix today.

**Comment by bhuisman1992** [comment]
Awesome!

**Comment by bhuisman1992** [comment]
I think the problems lies in the new field type added in 5.6.6: accordion. It is the same feature as the accordion plugin.. so thats why the website thinks there are two plugins installed. Disabling the addon plugin fixes the problem.. you can still use the accordion fields because of the new field type.

**Comment by bvdr** [comment]
This makes no usage for this addon plugin then :) 
Is it worth it to continue maintain it?

**Comment by blcarson** [comment]
No, I think you're good to retire it. Thank you for all the hard work!
